tests/ctst/features/azureArchive.feature

```diff
@@ -0,0 +1,129 @@
+Feature: Azure Archive
+
+    @2.7.0
+    @PreMerge
+    @Flaky
+    @AzureArchive
+    Scenario Outline: Archive objects when timeout is reached
+    Given a "<versioningConfiguration>" bucket
+    And a transition workflow to "e2e-azure-archive" location
+    And an object "obj1" of size <objectSize> bytes
+    And an object "obj2" of size <objectSize> bytes
+    Then the storage class of object "obj1" must become "e2e-azure-archive"
+    And the storage class of object "obj2" must become "e2e-azure-archive"
+    And manifest containing object "obj1" should "contain" object "obj2"
+    And manifest access tier should be valid for object "obj1"
+    And tar access tier should be valid for object "obj1"
+
+    Examples:
+        | versioningConfiguration | objectSize |
+        |           Non versioned |        100 |
+        |               Versioned |        100 |
+        # |               Suspended |        100 |
+
+    @2.7.0
+    @PreMerge
+    @Flaky
+    @AzureArchive
+    Scenario Outline: Archive 0 byte objects
+    Given a "<versioningConfiguration>" bucket
+    And a transition workflow to "e2e-azure-archive" location
+    And an object "obj1" that "exists"
+    And an object "obj2" that "exists"
+    And an object "obj3" that "exists"
+    Then the storage class of object "obj1" must become "e2e-azure-archive"
+    And the storage class of object "obj2" must become "e2e-azure-archive"
+    And the storage class of object "obj3" must become "e2e-azure-archive"
+
+    Examples:
+        | versioningConfiguration |
+        |           Non versioned |
+        |               Versioned |
+        # |               Suspended |
+
+    @2.7.0
+    @PreMerge
+    @Flaky
+    @AzureArchive
+    Scenario Outline: Respect maximum number of objects per archived Tar
+    Given a "<versioningConfiguration>" bucket
+    And a transition workflow to "e2e-azure-archive" location
+    And an object "obj1" of size <objectSize> bytes
```

Comment:
could we add another parameter to this rule:

> And <count> objects "obj" of size <objectSize> bytes

where we could either run with multiple counts (→ see different behaviors), or at least avoid repeating 5 times the same step/condition ?